tests/integrationTests/tests/c_failure_messages/validation/grade.txt_alternate

```diff
@@ -1,8 +1,8 @@
-Testcase  1: Compilation                                          5 /  5  
-Testcase  2: NORMAL GRADING:  (hiding the details)               10 / 10  
-Testcase  3: BEHIND THE SCENES:  raw output                       3 /  5  
-Testcase  4: BEHIND THE SCENES:  checking the sorted output       5 /  5  
-Testcase  5: BEHIND THE SCENES:  checking the second and third words   5 /  5  
-Testcase  6: Submission Limit                                             
+test01: Compilation                                          5 /  5  
+test02: NORMAL GRADING:  (hiding the details)               10 / 10  
+test03: BEHIND THE SCENES:  raw output                       3 /  5  
+test04: BEHIND THE SCENES:  checking the sorted output       5 /  5  
+test05: BEHIND THE SCENES:  checking the second and third words   5 /  5  
+SubmissionLimit: Submission Limit                                             
```

Comment:
do we somewhere check/prevent using this id for an instructor-written testcase?
(would be better not to have this limitation at all, but I don't have a better suggestion right now)

Reply:
At the moment, instructors can't provide there own testcase ids. There is a comment in the code where this value is set (requested in your previous review) which lists adding a check for duplicate ids as a todo.